### PR TITLE
added mangareader.to

### PR DESCRIPTION
**Pull request by LiReNa00**

### CODE REVIEW

**Comment by LiReNa00** [comment]
Hmm my other pull request was merged right away why this one taking time hmmm

File: README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ A curated list of all things anime - streaming/downloading/reading/tracking/foru
 * [anime-loads.org](https://www.anime-loads.org/) - Contains German Dubs/Subs
 * [animepahe.com](https://animepahe.com/)
 * [anime-planet.com](https://www.anime-planet.com/)
+* [animixplay.to](https://animixplay.to/)
```
Comment (anshumanv) [inline README.md:57]: this doesn't work

**Review by anshumanv — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LiReNa00) [reply README.md:57]: wdym it doesn't work hmm

Comment (LiReNa00) [reply README.md:57]: this site works for me in India tho

Comment (anshumanv) [reply README.md:57]: then I think just blocked in Japan, @toprak can you check?

Comment (anshumanv) [reply README.md:57]: then I think just blocked in Japan, @toprak can you check?

Comment (toprak) [reply README.md:57]: @anshumanv sorry for very late reply, i have been busy with school thing anyways it works for me

**Review by toprak — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (LiReNa00) [reply README.md:57]: That was quite a time taking thing 🤧

Comment (toprak) [reply README.md:57]: @LiReNa00 yes sorry about that i got many problems also too busy with school i dont have life anymore :D

Comment (LiReNa00) [reply README.md:57]: @toprak that is very relatable 🤧

**Pull request merged**